### PR TITLE
Make some reorganizations in the chapter.

### PR DESCRIPTION
**Pull request by jb-brelot-nxp**
THe idea is to move any chapter relative to ABI to Appendix

### CODE REVIEW

**Comment by james-ball-qualcomm** [comment]
Can we move chapters 3 & 4 (M-mode and S-mode CSRs for configuring CLIC) under chapters 5 & 6 (smclic and ssclic respectively)? We could have a heading in smclic/ssclic for direct CSRs and another for indirect CSRs. Seems strange that the smclic and ssclic chapters that list CSRs in their privilege mode don't include all their CSRs. I'm guessing the current organization is an artifact of having memory-mapped configuration in older CLIC versions.

**Comment by christian-herber-nxp** [comment]
Update the change log

**Comment by jb-brelot-nxp** [comment]
Updated version of the pull request :
chapter 3 and 4 have been moved
chapter 11 is transformed as a NOTE in 5.5
Moving chapter 12 in appendix
Remove "Appendix :" typo

**Review by christian-herber-nxp — APPROVED** [review]
lgtm

File: src/clic.adoc [hunk 1]
```diff
@@ -796,6 +659,81 @@ interrupt trigger when using accesses of {nxti} that include writes.  A trigger
 The `interrupt_trap_enable` control bit is read-write to enable/disable this
 interrupt trigger.  A trigger is signaled to the debug module if an interrupt trap is taken and the interrupt code matches a `clicinttrig[__i__]`.interrupt_number and the associated `clicinttrig[__i__]`.interrupt_trap_enable is set.
 
+=== M-mode CLIC Register Access via Indirect CSR Access
```
Comment (james-ball-qualcomm) [inline src/clic.adoc:662]: Can you rename this to "Indirect Access M-mode CSRs"?

File: src/clic.adoc [hunk 2]
```diff
@@ -1415,6 +1353,75 @@ The {ret} instruction does not modify the {cause}.{pil} field in {cause}.
 == ssclic S-mode CLIC extension
 The ssclic extension depends on the smclic extension.
 
+=== S-mode CLIC Register Access via Indirect CSR Access
```
Comment (james-ball-qualcomm) [inline src/clic.adoc:1356]: Can you rename this to "Indirect Access S-mode CSRs"?

File: src/clic.adoc [hunk 3]
```diff
@@ -3016,7 +3017,7 @@ ID  Interrupt
 6+  local
 ----
 
-=== CLIC-mode interrupt-map recommendation for single-hart M/S/U systems without N-extension with no PLIC/APLIC:
+=== CLIC-mode interrupt-map recommendation for single-hart M/S/U systems with no PLIC/APLIC:
```
Comment (james-ball-qualcomm) [inline src/clic.adoc:3020]: Thanks!

**Review by james-ball-qualcomm — CHANGES_REQUESTED** [review]
I suggested some changes to the indirect access CSR heading title. 
Can we make parallel naming changes to the CSRs that we already in the smclic/ssclic chapters? Namely that would be change "CLIC CSRs" to "Direct Access M-mode CSRs" (in smclic chapter) and "Direct Access S-mode CSRs" (in ssclic chapter).

File: src/clic.adoc (hunk 1 above)
Comment (christian-herber-nxp) [reply src/clic.adoc:662]:
```suggestion
=== Indirect Access M-mode CSRs
```

File: src/clic.adoc (hunk 2 above)
Comment (christian-herber-nxp) [reply src/clic.adoc:1356]:
```suggestion
=== Indirect Access S-mode CSRs
```

**Review by james-ball-qualcomm — APPROVED** [review]

**Pull request merged**